bindings/jaxrs/src/main/java/org/commonjava/indy/core/bind/jaxrs/ContentAccessHandler.java

```diff
@@ -307,6 +308,13 @@ public Response doHead( final String packageType, final String type, final Strin
 
                     responseHelper.setInfoHeaders( builder, item, sk, path, true, contentType,
                                     httpMetadata );
+    
+                    if(!path.endsWith("/")) {
```

Comment:
You had this in ResponseHelper before...what prompted you to move it into ContentAccessHandler? Was it causing problems with some other REST class?

Reply:
Yes , some of integration test have made errors [RecalculateMissingChecksumForTrackingRecordTest] on build  so that's way i move it it here in ContentAcessHandler.